HW2/P5/driver.py

```diff
@@ -13,12 +13,27 @@
 from animator import Animator
 from physics import update, preallocate_locks
 
+# from the wiki page on z-order curve
+def cmp_zorder(a, b):
```

Comment:
Nice choice of z-order. I developed one scheme myself but I think it is no better than z-order